region/lowernorfair/east.json

```diff
@@ -1953,31 +1977,79 @@
           "to": [
             {
               "id": 1,
-              "requires": null,
+              "requires": [
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                "canUsePowerBombs",
```

Comment:
I don't understand the PB cost here

Reply:
In order to avoid taking damage from the multiviolas there are really no other good, fast and reliable option for the heatruns. However I guess I could stick it into the HeatRun portion.

Reply:
Oh, I see. Do you think tanking the damage should be an option? It doesn't seem like it would be that costly if you had Gravity Suit.

Reply:
The turnout is usually 3x Multiviola and 2x Desgeega hits. It's survivable, even suitless with a lot of health. Putting it in.

Reply:
This looks like it will complicate the link's reqs significantly though. What's your opinion on adding a junction at the bottom right to simplify this?